Tests/PythonKitTests/NumpyConversionTests.swift

```diff
@@ -0,0 +1,42 @@
+
+import XCTest
+import PythonKit
+
+
+class NumpyConversionTests: XCTestCase {
+
+  static var numpyModule: PythonObject?
+
+  override class func setUp() {
+    super.setUp()
+    PythonLibrary.useVersion(3, 7)
```

Comment:
Could we generalize this test to Python 3?

`PythonLibrary.useVersion(3)`

Reply:
Fixed in 79c5933